src/styles/index.scss

```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@import '/src/styles/utils/variables';
```

Comment:
Ensure that the file `/src/styles/utils/variables.scss` exists and contains the necessary variable definitions, such as `$main-text-color` used in `body.scss`.